src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/ResourceQueriesQueryString.feature.disabled

```diff
@@ -0,0 +1,63 @@
+Feature: Query String handling for GET requests for Resource Queries
+
+        Background:
+            Given a school exists
+                  | schoolId  | nameOfInstitution      | gradeLevels       | educationOrganizationCategories |
+                  | 255901001 | Grand Bend High School | [ "Tenth grade" ] | [ "School" ]                    |
+              And there is a valid AcademicWeek
+                  | weekIdentifier | beginDate  | endDate    | totalInstructionalDays |
+                  | Week One       | 2024-05-15 | 2024-05-22 | 2                      |
+
+        @ignore
+        Scenario: Ensure clients can GET information when querying by valid date
+             When sending a GET request to "/ed-fi/academicWeeks?beginDate=2024-05-15"
+             Then it should respond with 200
+              And response body is
```

Comment:
This should be `And the response body is`

Reply:
Done